tests/util/mod.rs

```diff
@@ -38,3 +38,26 @@ impl AsRef<Path> for &DBPath {
         &self.path
     }
 }
+
+#[allow(dead_code)]
+pub fn pair(left: &[u8], right: &[u8]) -> (Box<[u8]>, Box<[u8]>) {
+    (Box::from(left), Box::from(right))
+}
+
+// Use macro rather than functions so that on failure we get line number where
+// the macro was instantiated rather than line number inside of a function.
+#[allow(unused_macros)]
+macro_rules! assert_iter {
```

Comment:
Can this macro be replaced with a function with the `#[track_caller]` attribute?

Reply:
Perfect. Done.